src/crwimage.cpp

```diff
@@ -92,10 +92,10 @@ void CrwImage::writeMetadata() {
   CrwParser::encode(blob, buf.c_data(), buf.size(), this);
 
   // Write new buffer to file
-  auto tempIo = std::make_unique<MemIo>();
-  tempIo->write((!blob.empty() ? &blob[0] : nullptr), blob.size());
+  auto tempIo = MemIo();
```

Comment:
Good catchs in this commit 👏 . Is this something automatically detected by clang-tidy?

I would only suggest to take a step more here. I think it is more intuitive to write `MemIo tempIo;` (Here and in all the other cases)

Reply:
Nope. Saw it while browsing the code.